.changeset/no_magic_numbers.md

```diff
@@ -0,0 +1,12 @@
+---
+"@biomejs/biome": minor
+---
+
+Added [noMagicNumbers](https://github.com/biomejs/biome/issues/4333) rule.
+The rule detects and reports the use of "magic numbers" — numeric literals that are used directly in code without being assigned to a named constant.
+
+## Example
```

Comment:
```suggestion
Example:
```

It's best to not use headers in changesets, otherwise they break upstream services such as renovate.

If you want to use them though, they should be `####`. In this case I think it's not needed